draft-ietf-ohai-ohttp.md

```diff
@@ -644,10 +644,13 @@ POST request to the oblivious proxy resource.  This request MUST only contain
 those fields necessary to carry the encapsulated request: a method of POST, a
 target URI of the oblivious proxy resource, a header field containing
 the content type (see ({{media-types}}), and the encapsulated request as the
-request content.  Clients MAY include fields that do not reveal information
-about the content of the request, such as Alt-Used {{?ALT-SVC=RFC7838}}, or
-information that it trusts the oblivious proxy resource to remove, such as
-fields that are listed in the Connection header field.
+request content. In the request to the oblivious proxy resource, clients MAY
+include additional fields. However, those fields MUST be independent of the
+encapsulated request and MUST be fields that the oblivious proxy resource will
+remove before forwarding the encapsulated request towards the target, such as the
+Connection or Proxy-Authorization header fields {{?SEMANTICS=RFC9110}}.
+Content-Length is an exceptional header that clients MAY include in requests
+to the oblivious proxy since it does not reveal any new information to the proxy.
```

Comment:
```suggestion
```

I realize that this statement is true, but it is also distracting.  Content-Length is not as much a field as a framing mechanism in HTTP.  It has to match the length of the content. So while it isn't independent of the content, it can't be, nor does it convey new information about the content, except to the extent that it is used for framing and is therefore necessary for the protocol to function at all.

Reply:
Agree

Reply:
I don't quite agree, but less text is better.